eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/EventsourcedProcessor.scala

```diff
@@ -113,7 +120,9 @@ trait EventsourcedProcessor extends EventsourcedWriter[Long, Long] with ActorLog
     case payload if processEvent.isDefinedAt(payload) =>
       val currentProcessedEvents = processEvent(payload)
       if (lastSequenceNr > processingProgress)
-        processedEvents = processedEvents :+ currentProcessedEvents.map(createEvent(_, lastHandledEvent.customDestinationAggregateIds))
+        processedEvents = processedEvents :+ currentProcessedEvents.map { e =>
+          postProcessDurableEvent(createEvent(e, lastHandledEvent.customDestinationAggregateIds))
```

Comment:
Good arguments @danbim, thanks. We definitely need a better specification which `DurableEvent` fields should be allowed to change and which not. More importantly, both the actor-based and the stream-based event processors should enforce that specification consistently. I created a separate ticket for a common event processing model (#376) so that we can cover that later.

In context of this ticket, I'm fine with your initial proposal that is based on the "garbage in, garbage out" assumption. Let's keep it as an (undocumented) expert feature and document the work results of #376 later. It anyway closely matches what can be done with the stream-based processor at the moment.

Regarding the test, adding one that only changes `customDestinationAggregateIds` should be sufficient.